### PR TITLE
Reduced HP regen to 70% current value

### PR DESCRIPTION
**Pull request by Regularitee**
```SUMMARY: Balance "Reduced HP regeneration to 70% previous value"```

Reduced player/NPC natural regeneration to ~~30%~~ 70% current value. 

Majority of players in https://github.com/CleverRaven/Cataclysm-DDA/issues/25547 concur current value is too high. This is a first-pass balance attempt, and may be followed up with another depending on player feedback.

### CODE REVIEW

**Comment by MT-Arnoldussen** [comment]
how does this affect bandages and the various healer traits?

**Comment by woutershep** [comment]
You forgot to change the info text to match it.

**Comment by nexusmrsep** [comment]
@Regularitee I've added a summary line for Jenkins rebuild

**Comment by Regularitee** [comment]
@MT-Arnoldussen You can inspect the bandage (and all other healing) mechanics in the data/jason/effects.json

I intentionally did not increase bandage potency to compensate. Since (makeshift) bandages are effectively unlimited in availability, any increase in potency to compensate could easily be abused to circumvent the natural healing nerf.

**Comment by MT-Arnoldussen** [comment]
@Regularitee I don't think bandages should be buffed to compensate for this , but I do think they should not become worse due to it, and I am under the impression that #23757 made bandages scale with sleep healing rate.

**Comment by Xhuis** [comment]
This will make Poor Healer incredibly worse to take given its -4 point cost and Imperceptive Healer effectively disable sleep healing entirely. Please consider modifying their point costs in light of this.

**Comment by Regularitee** [comment]
I would, ideally, like to get some feedback on how this impacts healing across the board before tinkering with other aspects of healing. Otherwise changes to other aspects will be at least partially guesswork as to what is ideal.

That said, the code indicates these traits are a healing modifier rate rather than a flat increase or decrease (unless I'm misinterpreting it). So their proportional effectiveness will be the same compared to normal humans or fast healers. I'm not entirely certain that warrants a cost reduction -- and if it does, I'm not certain how much exactly should this be. Especially since we would likely have to give faster healing mutations an inverse cost increase.

This is the sort of thing that I think deserves its own issue and some widespread player input before tinkered with.

**Comment by John-Candlebury** [comment]
I don't really see much agreement for this in #25547

In my opinion this isn't really needed, it slows gameplay greatly and the balance concerns are not fully adressed, you need to at least present new numbers of what are the expected heal rates using health kits and the related trait and skill combinations.

Otherwise you are just changing stuff through feelings and there's no guarantee that what you propose is more properly balanced.

**Comment by Xhuis** [comment]
I am inclined to agree. A single issue doesn't represent the majority opinion, and this is a ***massive*** change with huge repurcussions across the entire community. In fact, a comment (describing it being a side effect of the blob) has more votes up than the actual issue itself.

If this goes over poorly, you will be directly responsible for causing a game-shattering change until it's adjusted and reverted, and I know from experience being that person that it's not something you want to do. I urge you to take more opinions into consideration through reddit, discord, etc. before you make this.

**Comment by Regularitee** [comment]
@John-Candlebury 
I explicitly asked for exact recommendations for an exact number on what the new healing factor values should be a few days ago. The only person to give a precise number was someone who recommended a 10% healing factor, no one else did. Neither for an increase or decrease. I suspect that's because it's difficult to simply come up with an abstract number and imagine how it could apply to every possible play-style and scenario. We have to start _somewhere_, and adjust further from there. 

@Xhuis 
I think it's disingenuous to call this a "**massive**" or "game-shattering change"; this change does not introduce a new system, nor does it remove an existing system. It will simply slow down the pace of healing. One or two extra days of bed rest _each time a player is brought to the brink of death_, will not be an apocalyptic game-breaker. The only way it would have such an impact is if the player is being brought to the brink of death on a daily basis.

And although you bring up reduced-healing traits, I could make a similar argument in turn: have you considered the relative uselessness of enhanced-healing traits when a player can already fully restore health on a daily basis? In addition to health-regenerating medicine being render useless for the same reason.

**Comment by Xhuis** [comment]
Have you played with the Poor Healer or Imperceptive Healer traits before?

Since their release, I've used them since I enjoy the challenge. Poor Healer, which is a reflection of what this change will be - one-third healing - makes healing far longer, and makes damage mean more. Imperceptive Healer is slow enough that getting damaged at all takes multiple days to heal even with high first aid and medical supplies.

I think this *is* a massive change, because it severely reduces the main way of restoring health through sleep. It's the same reason I think that both traits should have point increases, because Poor Healer with this change is 10% of current healing, and Imperceptive Healer is roughly *3% of current.* That isn't worth only 8 points to me.

Fast Healer does, additionally, have a place. It takes multiple sleep sessions to heal grievous wounds at default; Fast Healer expedites that, and grants a very slow healing while awake as well. So, yes - I do think they have a place.

Bandages, gauze, etc. have a place with the current system for the same reasons; if your wounds are severe, applying medicine will help them heal fast enough that you can be up and ready to fight again by the next day. They are also the primary method of regenerating health with the slow-healing traits, because sleep healing is so slow on its own that not using bandages is almost equivalent to not healing at all.

**Comment by Regularitee** [comment]
Um... about that. Your numbers are wrong.

I'm not sure you realize this, but the positive healing traits increase `healing_awake` and `healing_resting`... but the negative healing traits reduce only `healing_resting`. Meaning the penalty only applies while asleep, and therefor only about ~1/4th of game time (on average? Assuming 6 hours sleep a day). To give you an example:

        "type" : "mutation",
        "id" : "FASTHEALER",
        [...]
        "healing_awake" : 0.2,
        "healing_resting" : 0.5`


        "type" : "mutation",
        "id" : "SLOWHEALER",
        [...]
        "healing_resting" : -0.25

So mathematically, the worst case scenario (-0.9 sleep healing), only applying to 1/4 of game time, actually turns into -0.21 healing overall. 

Additionally, I would remind you of one of your own previous arguments earlier: 
>The Poor Healer and Imperceptive Healer traits already exist for players who feel that sleep healing is too forgiving. I see little reason to baseline it.

Inversely, why can't I in turn state, "The Fast Healer and Very Fast Healer traits already exist for player who feel that sleep healing is too little."? These are purely optional traits, as you yourself have pointed out.  If someone opts into taking a trait that reduces healing, of course they're going to have awful healing. That's the whole point of the trait.

**Comment by Robik81** [comment]
> 1/4 of game time

I don't know what is that supposed to mean.

Normal character does not heal while awake at all. If you pick trait that reduces `healing_resting`, it reduces it for 1/1 of healing game time.

**Comment by Xhuis** [comment]
Characters do indeed not heal while awake by default without mutations or perhaps the Repair Nanobots CBM. There is also very large difference between rewarding points for a challenge trait, and nerfimg baseline healing to be what a trait that costs points gives.

Ultimately, I don't think any of what I'm saying will change what you end up doing, so I won't actively hinder you every step of the way, but I think this isn't a very healthy change for the game and I think most people would agree if you sampled a larger audience.

**Comment by Kelenius** [comment]
Do you have any rationale for this change other than "I feel that the healing is too fast"?

**Comment by Night-Pryanik** [comment]
> Do you have any rationale for this change other than "I feel that the healing is too fast"?

This rationale is good enough. The only things OP should do are to back up his feelings with some numbers and "Expected vs Actual" points.

**Comment by Vasyan2006** [comment]
To clarify:
It will not change healing from bandages, medical items can be rebalanced in other json files.


Healing mutations will work the same way as before, as healing or rotting from mutations are multiplied by default health regeneration. Fast healer will get 50% more HP from sleeping for any game settings.
Using imperceptive healer trait to reduce healing is a bit tricky, because it is supposed to be a difficulty setting, but can be cancelled by any other healing mutation. Changing default value instead will scale all healing mutations.

**Comment by Robik81** [comment]
> It will not change healing from bandages

Are you sure about that?

https://github.com/CleverRaven/Cataclysm-DDA/blob/0d11af0be4818cf3787166157a2b31994b5ab9da/src/character.cpp#L2747

**Comment by Vasyan2006** [comment]
Yes, character with fast healer trait will receive twice more HP from bandages compare to slow healer character. Descriptions of healing mutations must be updated to make it obvious.

**Comment by MT-Arnoldussen** [comment]
I think healer traits affecting bandage healing is an issue on its own.

Even if this gets merged, sleep healing will still be very fast, and first aid nothing but a convenient tool if you don't have some books to read or crafting to do when you get injured.

Having bandages not work properly for the few characters that actually are in need of them just seems wrong to me.

**Comment by Marrim** [comment]
I'll just say that I have played for some time with player_healing_rate changed to 30% its original value. Otherwise it felt like healing items were completely unnecessary other than to stop bleeding / deep bites. Once I changed it, it was much more enjoyable to play.

As for the blurb that player healing rate is deliberately high because of the blob, well... 30% is still way better than real life.

**Comment by Vasyan2006** [comment]
Some math to complain about:
Default healing is 0.01 HP/turn or 6 HP/hour or 48 HP per 8 hours of sleeping. Or 57% of the HP pool.
Character will regenerate after one nigth.

After changes it will be 0.003 HP/turn or 1.8 HP/hour or 14.4 HP per 8 hours of sleeping. Or 17% of the HP pool.
Medicile will be required only after heavy combat.

Default healing after 10x reducing is 0.001 HP/turn or 0.6 HP/hour or 4.8 HP per 8 hours of sleeping. Or 5.7% of the HP pool.
Medicile will be required for every wound worse than scratch.

**Comment by AbsalomAchitophel526** [comment]
Bandages were already nigh useless, and this change make it worse (Because bandage healing is based on sleep healing, and is already quite low.)  As for the comment about makeshift "bandages," they can only be applied once per 12-14 hours (Because they can only go up to a (Bandaged(2))status effect,) and provide around 5 HP if maintained for around 24 hours.

**Comment by MT-Arnoldussen** [comment]
I think this would be a good change IF bandage healing rate is decoupled from sleep healing rate. 

If you think that makes First Aid too strong because of how common makeshift bandages are, then alter makeshift bandages. Actual bandages (in combination with good skill) should noticeably heal you, even if you have Imperceptive Healer.

Actually, it should heal you *especially* if you have Imperceptive Healer.

**Comment by Xhuis** [comment]
Agreed about bandages being independent. Making bandages useful with this new change makes it from a large nerf for its own sake into something that could stay least be construed as a balance shift.

**Comment by Kelenius** [comment]
Why should they be independent? Bandages IRL are not magical things that you put on your scratches to heal them. Healing rate from the healing items should absolutely be tied to your regeneration speed!

I'm still against this change. There's no reasoning behind it, like IRL data or even a proper justification for balance concerns. You feel like the healing is too fast - okay, good for you. I don't. Change your game's settings if you want.

You say that a "majority of players" agrees with you. Let's count - sfsworms, Regularitee, teapot156, nexusmrsep... four people agree with you. Four! And there are several people in that very thread that don't want this change. This PR currently has 4 thumbs up and 9 down. What majority are you talking about?

This PR simply gives every character a negative trait worth 4 points on a whim and for arbitrary reasons.

**Comment by MT-Arnoldussen** [comment]
>Healing rate from the healing items should absolutely be tied to your regeneration speed!

Because it makes healing items pointless. 

Now they work when you don't need them, and they don't work when you do.

Imperceptive Healer used to be a trait that made you go craft or hunt medical supplies because otherwise you'd barely heal. Now it's just a trait that makes healing take longer because bandages no longer work on you.

If bandages have to be scaled with healing rate, then I would suggest just removing them. Rename First Aid to BIonic Installation and stop pretending the skill does anything other than that.

**Comment by nexusmrsep** [comment]
Previously you slapped some bandages to magically get on your feet, now you don't  even need that as you regain hp faster then you can lose them, so no need to guess why rebalancing is unpopular. Exaggerated but you know what I mean by that.

Rant aside, seriousness follows:
Argument that you can set it lower for your game is invalid, as you can as well up the value in yours, if you want, instead of saying that unbalanced value should be the new standard. 

Argument that feature B dependency on feature A should prohibit rebalancing broken feature A is also invalid. Instead both A and B should be balanced accordingly.

IMHO base healing is too fast. Remember that we are talking about the untreated wounds so whatever the new balance should be, the current heal-overnight effect is far from desired. System introduced by Vasyan, while not perfect, has finally introduced some sanity in form of wound healing slowly by themselves and medical items aiding in many ways to the process. This system has in mind that base healing when you don't do absolutely anything with your wounds should be slow, and items like bandages, disinfectants should become highly important, even if makeshift. Ideally you want to bandage and disinfect every wound, every time, to get proper healing rate. In other words thing of the base rate as debuffed, and treated rate as a target, desired rate.

What I'm saying here is that:
- bandages and disinfectants should boost healing rate by more significant value
- base healing rate should be lowered
- bandaged and disinfect wounds healing rates should resemble current base rates
- every trait, cbm, etc. should be **afterwards** rebalanced to new base factors above at they come second, not first.

There should also be more intermediate disinfectants. I'm actually thinking of adding recipe for soaped water as a weak disinfectant.

And bandages and disinfectants healing boost should not be dependant on/linked  wirh sleep rate in a way so they work only when sleeping. Haven't gone through every line of code to confirm it, but they should give independent bonus and sleep would then have independent overall modificator. (Base + bandage + disinfectant ) * sleep modifier.

This will never be near RL even if you tweak it down yourself, but for now it'd be more towards sanity then it is. One is for sure - if you tweak one knob others must follow unless scalability is/was assured throughout the system based on one/two values, which I believe is not the case.

For future development I'm looking forward to independent wounds system instead of a sack of HP's, if someone would code it.

**Comment by MT-Arnoldussen** [comment]
> (Base + bandage + disinfectant ) * sleep modifier.

Why apply the sleep modifier to bandage healing?

The slow/poor/imperceptive healer traits should be more than just increased tediousness, which is what they are with such a system. If they do *not* affect medical healing, they offer interesting gameplay scenarios in that you want to get medical supplies to heal. If they make bandages worthless, then you just have the current scenario where they do nothing but make you spend more time spamming the sleep button.

**Comment by Robik81** [comment]
> Why should they be independent? Bandages IRL are not magical things that you put on your scratches to heal them. Healing rate from the healing items should absolutely be tied to your regeneration speed!

Funny how you argue for IRL against independent healing rate for medical items. And throw IRL argument for slower healing under the bus in the same post.

If IRL is sole purpose of everything and all, we should start with defaulting wild life only.

So, to answer your question, they should be independent, because it makes sense from game play perspective, and because traits themselves suggests the alter only natural regeneration from sleeping, not nerf medical items.

**Comment by Marrim** [comment]
Cata is a simulationist game.
Bandages serve to help the body repair itself.
IRL argument for slower healing = valid.

**Comment by Robik81** [comment]
I could make the shit up and say that the medical condition of these traits is, that it specifically hampers regeneration of improperly treated wounds.

**Comment by Vasyan2006** [comment]
>  (Base + bandage + disinfectant ) * sleep modifier.
But this is exactly how it works, right?

**Comment by Kelenius** [comment]
I would be totally for moving to a more realistic system where individual wounds are tracked and heal realistically. Simply slowing healing down _does not_ make the game more realistic. You move from one arbitrary point to another. You'd need to find some IRL data to justify this change, and you'll never find it, because the current system isn't realistic in the first place.

Bandages are useful for stopping bleeding, preventing contaminants from getting into wounds, etc, etc, etc, but they don't make the wounds heal faster on their own. It's still your body doing the work healing it. So yes, again, it makes sense that bandages speed up the healing speed proportionally to your normal healing rate.

**Comment by Robik81** [comment]
> Simply slowing healing down _does not_ make the game more realistic.

It does though? Healing is slower in RL than in the game, thus making it slower in game would make the game more like RL, without any other change that would make it even more like RL.

**Comment by MT-Arnoldussen** [comment]
If the First Aid skill is not supposed to noticeably heal your character, then I propose just removing the First Aid skill. It's a completely pointless skill at this point, and it's better not to give both old and returning players the impression that it does things it doesn't.

Just move the CBM installation chance to Computers, and the various recipes to Fabrication and Cooking (and maybe Tailoring).

**Comment by Marrim** [comment]
Bandages already  do heal individual wounds rather than simply increase healing rate.
If you were wounded for 20 hp, then got bandaged, then got wounded for 20 more hp and then went to rest, then bandages would only be effective for healing 20 hp out of the 40 you lost.

**Comment by aawilson** [comment]
If the desire is that bandages have more of a gameplay role, perhaps have reasonably default medical items (balanced to some whatever-arbitrary level of First Aid) bring Slow Healers/Imperceptible Healers up to 100% and 50% (respectively) of the healing rate of someone without those, but put a stepdown on the impact as healing rate increases that tone down the impact for normal/augmented healers (keeping normal healers the same, for now, and keeping augmented healers from blowing through the roof with the new numbers). I don't think (and have argued in the other thread) that straightforwardly reducing healing time is a good solution to the perceived problem, but "the old speed of healing, just with material requirements" sounds more palatable, especially in the context of making First Aid more than just a "meet threshold to install cybernetics" skill. This would also be easier than completely overhauling damage, which sounds pleasant but, currently, theoretical and thus not worth discussing until someone commits time to it.

**Comment by dissociativity** [comment]
I think this is perfectly fine until we can track individual wounds, the current system of going from near death to healthy in a night is ridiculous to say the least.

**Comment by shabang50** [comment]
As a player, i'm against pinning healing rate to a fixed point. Is it possible to include heal rate (and sleeping heal rate) as a modifiable option in the world settings?

**Comment by MT-Arnoldussen** [comment]
@shabang50 you can take the traits. you can edit the amount of points you can for worldgen if you don't want it to earn or cost you points.

**Comment by Regularitee** [comment]
If I may add something:

The focus of the discussion seems to be in large part "This should change" or "This shouldn't change!" This seems to create a misleading and false dichotomy that the only choices are between a 30% and 100% healing. Instead, it would be a lot more constructive if people were suggesting specific numbers if they agree with the current value. I can't help but wonder if some of the opposition are not purely protesting the healing itself, but its specific value. This was an issue in the previous thread where everyone had an opinion, but only one person volunteered a specific value. 

**If you do object, please make it clear whether _how_ you object**. Do you want to retain the 100% value, and will accept nothing else? Or would you find a reduction of healing to values like 90%, 80%, 70%, 60%, 50%, or 40% more acceptable?

To help facilitate this discussion, here's a breakdown of healing rates. Note:
- All values assume 100 player speed
- All values assume 14400 turns per day (as per above assumption, unless I got the number wrong)
- All values are only default rest healing. They ignore medicine or `healing_awake` factors.
- Values for healing are HP. All are rounded to nearest whole number.
- 100% represents the current value without changes

Sleep time | 100% regen | 80% regen | 60% regen | 40% regen | 20% regen
--|--|--|--|--|--
24 hours | 144 | 115 | 86 | 58 | 29
18 hours | 108 | 86 | 65 | 43 | 22
12 hours | 72 | 58 | 43 | 29 | 14
6 hours | 36 | 29 | 22 | 14 | 7

Remember, this is only unaugmented healing. You can reasonably assume on average, the actual healing rates will be somewhat higher due to use of bandages.

**Comment by Xhuis** [comment]
Assuming a general sleep time of 12 or 18 hours (I have never gone a full 24 hours sleeping the entire time, to my knowledge) the current values regenerate limbs to roughly full. That's how it's been intended for a long time up until this point, and even has a fluff explanation.

If you must reduce it, I think around 80% is the best place to put it so that grievous wounds remain a problem for more than one day but aren't still life-threatening even after a full night's rest.

**Comment by MT-Arnoldussen** [comment]
I think any decrease of sleep healing is unacceptable if it also changes bandage healing with it, which is what would happen if this PR got merged.

Otherwise I think any decrease up to 30% or so would probably be fine.

**Comment by G3Kappa** [comment]
I'd go for 60% to 80%. But another option is scaling this factor **non-linearly**, so that more grievous wounds take more time to heal. It makes sense, it's not hard to implement, and it would probably satisfy most people.

A 70% reduction from baseline feels too dramatic, but if the factor was lowered only slightly we'd then have enough data to see whether:

- Players like the change,
- The change works as intended,
- Game balance is not compromised.

Personally, I find wound-healing to be way faster than broken-limb-healing. In a sense, this means that as a player I tend to maximize damage before sleeping it off, because the real problems only arise when you need a splint.

So, to recap, my proposal would be to:

- Reduce baseline healing rate to somewhere between 60% and 80%;
- Scale the healing rate non-linearly in proportion to limb health.

**Comment by aawilson** [comment]
>This was an issue in the previous thread where everyone had an opinion, but only one person actually volunteered a specific number for the healing factor.

It's worth considering that a few of the people in that thread and this one who object to it are objecting to the change, in general, and thus the specific number of "100%" is implied. And I still think lowering HP regen focuses on the wrong part of the problem, but it's also worth noting that the specific suggestion of "make the healing rate such that a bandaged character at some average skill heals at 100% of the current non-bandaged rate" is also specific. If the _problem being solved_ is that it feels like First Aid is pointless since healing is too fast, then that is a great, minimal-impact change that can be used to evaluate more feelings on the subject. But if the _problem being solved_ is that it feels like characters don't spend enough time screwing around doing nothing interesting in their home bases, then it's very much worth asking whether that's actually a problem (that is, if it's something a lot of players agree with), or if it just "feels like" it should be that way for the sake of pretending this is something like real life.

>But another option is scaling this factor non-linearly, so that more grievous wounds take more time to heal. It makes sense, it's not hard to implement, and it would probably satisfy most people.

This solution will have the effect if typical scratches and cuts not taking appreciable time to heal, while forcing bedrest for grievous wounds, effectively imposing a "you almost died" tax on the player for pushing things too far. It's an interesting suggestion, far more so than linearly increasing all healing time across the board. It is worth noting that, whatever incentives already exist for very safe (and very boring) play, this will exaggerate them.

**Comment by Marrim** [comment]
In general, bandages are still useful without the change, because they allow you to heal without sleeping.  It's just that you can still do very well without ever using them, which kinda blows the suspension of disbelief -- even if there is an in-universe explanation.

For blunt trauma I could see bandages not making much of a difference. But cutting damage should probably apply some debuff that only goes away when the wound is bandaged or when it heals completely. Perhaps increased chance of bleeding upon further hits.

**Comment by davidpwbrown** [comment]
I feel that the problem of healing being more "realistic" is not solely down to sleeping rates and and bandage rates etc.
As others have intimated, an injury "type" system would make healing more nuanced, and therefore make bandages and sleeping more effective at certain injuries than others.
I was playing the game "Battle Brothers" recently, and the soldiers would get a cut or a bruise, or a puncture on different body parts etc , and each one had different recovery times and different debuffs, and perhaps this sort of variance would work well in a game such as this, where realism is a partial focus.
Simply saying "all injuries are now 10% slower to heal" is fixing a problem by hitting it with a sledgehammer.
In the short-term perhaps, a reduction to healing times might be the "good-enough" fix we need, but I also think that an eventual injury type system is the only way to balance gameplay and realism satisfactorily.

**Comment by Regularitee** [comment]
Alright, since discussion has slowed down significantly, there doesn't seem to be a whole lot of merit in waiting for feedback much longer.

I'm going to bump the change to 70% of current baseline healing. I think it's a good point because it's sufficiently large enough to be noticeable and gather feedback, but also sufficiently small enough not to significantly slowdown gameplay for those opposed.

Please note this change is not necessarily permanent, and will be revisited upon further feedback in the future.

**Comment by Xhuis** [comment]
I still don't support this change, but having it at 70% rather than 30% is more acceptable, at least until proper wounds can be implemented. Reluctant "okay" from me.

**Comment by kevingranade** [comment]
Based on your chart, it seems to be something like 40-50 hp healed per night with no medical attention.
What is that with full medical attention?

The thing that's still missing here is an argument for why this is the right number, or any kind of way to determine what the right number is.

**Comment by dissociativity** [comment]
I think the best compromise is 70% should be default, but it should scale down with how wounded the player is.

So situation A: player has lost 25% health, wounds aren't too heavy, can sleep it off.
Situation B, player has lost 50% health, healing is now at 50%, there are consequences for the risks they have taken and mistakes they have made, healing items should be pretty necessary and the player really shouldn't be able to sleep this off so easily.
Situation C: player has lost most of health, the impact of this is really showing, healing is at 30%, serious medical attention really makes sense here.

**Comment by Vasyan2006** [comment]
After using both bandages and disinfectant characters limb will be healed by 1.33 HP/hour while awake and 2.66 while asleep (8 HP and 16 HP per 6 hours). This rate will decay after 6 hours. Character with high first aid skill will get more intensity levels of healing effects and maximum possible healing rate will be twice higher (16 HP for awake and 32 HP for sleeping character per 6 hours).
This rates are only for healing effects. Natural regeneration is calculated separately. Traits and health stat will also change healing speed.
Using only one type of healing items (only bandages or only disinfectant) or using less effective makeshift items will decrease healing rate.

File: data/json/game_balance.json [hunk 1]
```diff
@@ -74,14 +74,14 @@
     "name": "PLAYER_HEALING_RATE",
     "info": "Set base player healing rate per turn. Default: 0.01",
```
Comment (woutershep) [inline data/json/game_balance.json]: You still haven't updated the info text as mentioned in the earlyer comment. The same for the info text below.

**Comment by dissociativity** [comment]
I think 30% is perfectly fair, but healing items should be increased to 12 or 9 hours over 6 so that they help a full night's rest.

**Comment by Regularitee** [comment]
> The thing that's still missing here is an argument for why this is the right number, or any kind of way to determine what the right number is.

There is no objectively correct number, that's the issue. It's just a subjective choice in playstyle; whether you want to incentive players to confront enemies (high regen), or hide from and avoid enemies (low regen).

Even if I could provide an iron-clad argument as to why my reasoning is superior (and I'm not saying it is), the fact is people are uncomfortable with drastic changes from what they are accustomed to. You can see examples of this during major UI overhauls with software or websites, and all the anger it arouses. So even if a more significant nerf is the end-goal (and I'm not yet sure it is), it would be better to do it gradually.

**Comment by kevingranade** [comment]
> There is no objectively correct number, that's the issue. It's just a subjective choice in playstyle; whether you want to incentive players to confront enemies (high regen), or hide from and avoid enemies (low regen).

No argument for correctness = no merge.
You're basically doubling down on, "We're going to keep tweaking this until the screaming stops", which is not a remotely reasonable way to go about this.
